pkg/apis/deployment/v1/actions.generated.go

```diff
@@ -759,7 +759,7 @@ func (a ActionType) Optional() bool {
 	case ActionTypeRenewTLSCertificate:
 		return false
 	case ActionTypeResignLeadership:
-		return false
+		return true
```

Comment:
it could be generated in a way where only optional actions returns true and default returns false, so it would be more legible.

Reply:
I will do it in PR if it is fine